qiskit_cold_atom/providers/cold_atom_job.py

```diff
@@ -135,6 +135,28 @@ def status(self):
 
         return status
 
+    def error_message(self):
+        """
+        Retrieve the error message from the backend.
+
+        Returns:
+            error: A string describing the error that happened on the backend.
+        """
+        status_payload = {"job_id": self.job_id()}
+
+        r = requests.get(
+            self._backend.url + "/get_job_status/",
+            params={
+                "json": json.dumps(status_payload),
+                "username": self.user,
+                "password": self.token,
+            },
+        )
+
+        error_message = r.json()["error_message"]
+
+        return error_message
```

Comment:
```suggestion
        return r.json().get("error_message", None)
```
This should be more robust: covers the case where there was no error or if the backend did not define one.

Reply:
I was wondering how best to do this, thanks.